onnxruntime_extensions/_hf_cvt.py

```diff
@@ -22,11 +22,20 @@ def __init__(self, tokenizer):
         self.tokenizer = tokenizer
 
     def bpe_tokenizer(self, **kwargs):
-        hf_gpt2_tokenizer = self.tokenizer
+        hf_bpe_tokenizer = self.tokenizer
+
+        if ("Fast" in str(self.tokenizer)):
```

Comment:
May just report an error for users to ask them use the slow version.

Reply:
Sounds good!